### PR TITLE
fix(quantic): add default checked sort criterion and fix label wrap in quanticSort

### PR DESCRIPTION
**Pull request by fbeaupreCoveo**
### Related Issues 
[SFINT-4005](https://coveord.atlassian.net/browse/SFINT-4005) and [SFINT-4006](https://coveord.atlassian.net/browse/SFINT-4006)

### Proposed changes
Re-opened this PR to fix two issues raised for the sortBy component:

- Make relevancy `checked` by default so that the user can see that results are sorted by relevancy, even before he makes a selection in the combobox
- Make the space for the sort combobox bigger so that label doesn't split into two lines.
Screen Shot 2021-07-30 at 11 15 39 AM

### CODE REVIEW

**Review by lbergeron — APPROVED** [review]
Looks good to me!

File: packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.css [hunk 1]
```diff
@@ -6,5 +6,7 @@
 
 .sort__header {
     color: #3e3e3c;
-    padding: 6px 6px 0px 0px;
+    padding-top: 6px;
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.css:9]: Should we be using the slds class for the paddings? Or is it necessary to be in px?

`slds-p-top_x-small` maybe? or thing like that?

**Review by erocheleau — APPROVED** [review]
Looks good, just 1 question posted.

File: packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.css (hunk 1 above)
Comment (fbeaupreCoveo) [reply packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.css:9]: I tried using slds classes for padding but vertically, xx-small had the label aligned just a little too high and x-small just a little too low, so I went with px instead. Horizontally, even xx-small starts with a pretty large gap between the label and the combobox

**Pull request merged**